attachment_image_resize/models/ir_attachment.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Quartile
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+IMAGE_TYPES = ["image/png", "image/jpeg", "image/bmp", "image/tiff"]
+
+
+class IrAttachment(models.Model):
+    _inherit = "ir.attachment"
+
+    resize_done = fields.Boolean()
+
+    def _postprocess_contents(self, values):
+        self = self.with_context(resize_target_model=values.get("res_model"))
```

Comment:
I guess we should update `resize_done`, but correct me if it's not the case.
```suggestion
        self = self.with_context(resize_target_model=values.get("res_model"))
        self.resize_done = True
```

Reply:
Initially, I was expected to just run cron job for existing records before we started using the feature of this module.
Now I realized there may be a chance of using the feature for some models after we already used this module and running cron job again will also run the records that are already resize but I think this case will not happen frequently and no impact on data capacity. Assigning` self.resize_done = True` in `_postprocess_contents` will skip the trigger for the above case(want to resize after the module has been in use for some time.) because this method will be called for all attachments.

Reply:
@AungKoKoLin1997 The design doesn't look to me very coherent as is. At least with my suggestion, `resize_done` acts as explained with the help text. If we keep the current design, we need to explain the usage of the field more clearly.

Reply:
I will go with your suggestion to stick with the design.